Cargo.toml

```diff
@@ -23,6 +23,9 @@ maintenance = { status = "experimental" }
 [dependencies]
 coresimd = { version = "0.0.3", path = "coresimd/" }
 
+[dev-dependencies]
+auxv = "0.3.3"
```

Comment:
We use the `auxv` crate in `coresimd` to test `getauxval` and in `stdsimd` to test reading from `/proc/self/auxv`.